coalib/bears/Bear.py

```diff
@@ -56,6 +73,7 @@ class Bear(Printer, LogPrinter):
     """
 
     LANGUAGES = ()
+    REQUIREMENTS = ()
 
     @classproperty
     def name(cls):
```

Comment:
commit message: I think you don't need to mention the doctests, docs and tests are normal so a reader would understand why you added docs and doctests ;)